src/main/resources/cache.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you test adding a custom `/cache.xml` on the class path - I assume that would override this one?